src/Resources/Webhook.js

```diff
@@ -40,7 +40,7 @@ module.exports = class Webhook {
    * @return {Promise.<Object>} response
    */
   update(data) {
-    return internal(this).client.put(`webhook/${internal(this).reference}/update`, { 'data': data });
+    return internal(this).client.put(`webhooks/${internal(this).reference}/update`, { 'data': data });
```

Comment:
the uri is just webhooks/{reference}

Reply:
that is correct!!! You should read the class constructor.